scripts/run_digit_recognition.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""
+Real time recognition of digits drawn in the air by hand.
+
+Usage:
+  run_digit_recognition.py [--camera_id=CAMERA_ID]
+                           [--path_in=FILENAME]
+                           [--path_out=FILENAME]
+                           [--title=TITLE]
+                           [--use_gpu]
+  run_digit_recognition.py (-h | --help)
+
+Options:
+  --camera_id=CAMERA_ID      Index of the camera to be used as input. Defaults to 0.
+  --path_in=FILENAME         Video file to stream from
+  --path_out=FILENAME        Video file to stream to
+  --title=TITLE              This adds a title to the window display
+  --use_gpu                  Use GPU for inference
+"""
+from docopt import docopt
+
+import sense.display
+from sense import engine
+from sense import feature_extractors
+from sense.controller import Controller
+from sense.downstream_tasks.digit_recognition import INT2LAB, LAB2THRESHOLD
+from sense.downstream_tasks.nn_utils import Pipe, LogisticRegression
+from sense.downstream_tasks.postprocess import PostprocessClassificationOutput
+
+
+if __name__ == "__main__":
+    # Parse arguments
+    args = docopt(__doc__)
+    camera_id = args['--camera_id'] or 0
```

Comment:
```suggestion
    camera_id = int(args['--camera_id']) or 0
```
Just found out why it wasn't working for me yesterday. With an explicit camera id passed in, this becomes a string and OpenCV doesn't digest it properly. I think we have to fix this across all scripts.